### PR TITLE
Safety: do not crash on (I)DIV

### LINKED ISSUE #561: Safety checker crashes on DIV and IDIV
The safety checker thinks that these instructions take 2 arguments, while they take 3. I do not know how to fix that properly.

### PR DESCRIPTION
**Pull request by vbgl**
Fixes #561.

### CODE REVIEW

**Comment by eponier** [comment]
I don't understand the changes to toEC.ml (but I don't know this file). The second commit interprets divisions as "top", correct?

File: compiler/src/toEC.ml [hunk 1]
```diff
@@ -988,14 +989,14 @@ module Leak = struct
...
-    | Pget (aa, ws, x, e) -> 
+    | Pget (aa, ws, x, e) ->
       assert (aa = Warray_.AAscale); (* NOT IMPLEMENTED *)
       let x = x.gv in
       let safe = 
         let (_s,option) = Mv.find (L.unloc x) env.vars in
         if option then Initai(ws, L.unloc x, e) :: safe 
         else safe in
-      in_bound ws x e :: safe 
+      in_bound x e :: safe
```
Comment (bgregoir) [inline compiler/src/toEC.ml:999]: I don't understand why the ws argument of in bound is removed.
stack u64[1] t;
x = t[u16 2];
I have the impression that this code will be rejected but it should not.

Comment (vbgl) [reply compiler/src/toEC.ml:999]: Let me drop the first commit that is unrelated and clearly bad.

**Review by eponier — APPROVED** [review]
We are imprecise about divisions, but this is clearly better than to crash.

**Comment by vbgl** [comment]
Now it is also precise. And it checks what should be checked.

**Review by bgregoir — APPROVED** [review]

**Pull request merged**